mbf_costmap_nav/src/mbf_costmap_nav/footprint_helper.cpp

```diff
@@ -111,6 +111,102 @@ void FootprintHelper::getLineCells(int x0, int x1, int y0, int y1, std::vector<C
   }
 }
 
+void FootprintHelper::supercover(int x0, int x1, int y0, int y1, std::vector<Cell>& pts)
+{
+  {
+    auto POINT = [&pts](int y, int x) {
```

Comment:
bad style. MACRO vs function in c++

Reply:
I didn't want to change the copy-pasted code for reference, but I will fix the style at least